### ISSUE TITLE
cf login issue: Credentials were rejected, please try again.

### ISSUE THREAD

**Issue by pranghos** [cloudfoundry/cli#2007]
cf login is not working even if the password has been provided. It is again asking for password. 

```
C:\McD\covid\cur>cf login -u passcode -p aLfKrLidQp
API endpoint: https://api.us-south.cf.cloud.ibm.com

Warning: Your targeted API's version (3.83.0) is less than the minimum supported API version (3.85.0). Some commands may not function correctly.


Authenticating...
Credentials were rejected, please try again.

Password:
```

**Comment by andrew-edgar** [comment]
@pranghos there is a known issue with the CF Cli 7 and IBM Cloud. 

Please see the issue here https://github.com/cloudfoundry/cli/issues/2007

Also the IBM public cloud has not yet been updated to the version of CloudController that supports CLI v7 and this will come shortly.

This is resolved in the "edge" binary and should be officially released soon.

**Comment by univ0298** [comment]
I think Andrew meant to refer to this issue: https://github.com/cloudfoundry/cli/issues/1964

The V7 CLI introduced a regression and hopefully the CLI team will release the fix for it that was mentioned 25 days ago in the above issue

**Comment by univ0298** [comment]
@pranghos you should see that login succeeds with the newly released CLI v7.0.2

https://github.com/cloudfoundry/cli/releases/tag/v7.0.2

**Comment by JenGoldstrich** [comment]
Thanks for helping answer this @andrew-edgar and @univ0298!

Like they said I think this was expected at the time, I am closing this now please feel free to re-open if it you feel I've closed it in error

Thanks!
Jenna
